localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+d_centaur_company;Centaur Company;Compagnie centaure;Zentauren-Kompanie;;Compañía Centauro;;;;;;;;;x
+d_centaur_company_adj;Centaur;centaure;zentaurisch;;Centauro;;;;;;;;;x
+d_farraki_company;Farraki Company;Compagnie farraki;Farraki-Kompanie;;Compañía Farraki;;;;;;;;;x
+d_farraki_company_adj;Farraki;farraki;farrakisch;;Farraki;;;;;;;;;x
+d_zandalari_company;Zandalari Company;Compagnie zandalari;Zandalari-Kompanie;;Compañía Zandalari;;;;;;;;;x
+d_zandalari_company_adj;Zandalari;zandalari;zandalarisch;;Zandalari;;;;;;;;;x
+d_korkron;Kor'kron;Kor'kron;Kor'kron;;Kor'kron;;;;;;;;;x
+d_korkron_adj;Kor'kron;kor'kron;kor'kronisch;;Kor'kron;;;;;;;;;x
 d_taunka_band;Winterhorns;;;;;;;;;;;;;x
 d_taunka_band_adj;Winterhorn;;;;;;;;;;;;;x
 d_vrykul_company;Drakujar;;;;;;;;;;;;;x
 d_vrykul_company_adj;Drakujar;;;;;;;;;;;;;x
-d_drakkari_band;Drakkari Band;;;;;;;;;;;;;x
-d_drakkari_band_adj;Drakkari;;;;;;;;;;;;;x
-d_wolvar_band;Wolvar Band;;;;;;;;;;;;;x
-d_wolvar_band_adj;Wolvar;;;;;;;;;;;;;x
+d_drakkari_band;Drakkari Band;Bande drakkari;;;Compañía Drakkari;;;;;;;;;x
+d_drakkari_band_adj;Drakkari;drakkari;;;Drakkari;;;;;;;;;x
+d_wolvar_band;Wolvar Band;Bande varleu;;;Compañía wolvar;;;;;;;;;x
+d_wolvar_band_adj;Wolvar;varleu;;;Wolvar;;;;;;;;;x
 d_tuskarr_band;Ulu'lep;;;;;;;;;;;;;x
 d_tuskarr_band_adj;Ulu'lep;;;;;;;;;;;;;x
 d_northern_company;Northerns;;;;;;;;;;;;;x
 d_northern_company_adj;Northern;;;;;;;;;;;;;x
 d_goblin_northrend_company;PSC ''Whitesteam'';;;;;;;;;;;;;x
 d_goblin_northrend_company_adj;Whitesteam;;;;;;;;;;;;;x
-d_frost_vrykul_company;Ymirjar;;;;;;;;;;;;;x
-d_frost_vrykul_company_adj;Ymirjar;;;;;;;;;;;;;x
-d_gorloc_band;Gorloc Band;;;;;;;;;;;;;x
-d_gorloc_band_adj;Gorloc;;;;;;;;;;;;;x
-d_snobold_band;Shobold Band;;;;;;;;;;;;;x
-d_snobold_band_adj;Shobold;;;;;;;;;;;;;x
-d_goblin_northrend_trade_fleet;Northern Trade Fleet;;;;;;;;;;;;;x
-d_goblin_northrend_trade_fleet_adj;Northern Trade;;;;;;;;;;;;;x
-d_vrykul_fleet;Ravagers;;;;;;;;;;;;;x
-d_vrykul_fleet_adj;Ravager;;;;;;;;;;;;;x
-d_frozen_sea_fleet;Frozen Sea Fleet;;;;;;;;;;;;;x
-d_frozen_sea_fleet_adj;Frozen Sea;;;;;;;;;;;;;x
-d_darkshore;Auberdine;;;;;;;;;;;;;x
-d_darkshore_adj;Auberdine;;;;;;;;;;;;;x
-d_azuremyst_isle;Azuremyst;;;;;;;;;;;;;x
-d_azuremyst_isle_adj;Azuremyst;;;;;;;;;;;;;x
-k_crystalsong_forest;Crystalsong;For�t du Chant de cristal;;;;;;;;;;;;x
-k_crystalsong_forest_adj;Crystalsong;Cristalline;;;;;;;;;;;;x
-k_sholazar_basin;Sholazar;Sholazar;;;;;;;;;;;;x
+d_frost_vrykul_company;Ymirjar;Ymirjar;;;Ymirjar;;;;;;;;;x
+d_frost_vrykul_company_adj;Ymirjar;ymirjar;;;Ymirjar;;;;;;;;;x
+d_gorloc_band;Gorloc Band;Bande Górloc;;;Compañía gorloc;;;;;;;;;x
+d_gorloc_band_adj;Gorloc;górloc;;;gorloc;;;;;;;;;x
+d_snobold_band;Snobold Band;Compagnie Frigbold;;;Compañía Snóbold;;;;;;;;;x
+d_snobold_band_adj;Snobold;frigbold;;;snóbold;;;;;;;;;x
+d_goblin_northrend_trade_fleet;Northern Trade Fleet;Flotte de Commerce du Nord;;;Flota Comercial del Norte;;;;;;;;;x

```

Comment:
pls add Nördliche Handelsflotte (or an equivalent, if existing)

Reply:
It will be done.